tests/huggingface/sagemaker_dlc_test.py

```diff
@@ -67,7 +72,7 @@ def run_test(args):
     pytest.param("bigscience/bloom-560m", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
     pytest.param("EleutherAI/gpt-neox-20b", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
     pytest.param("google/flan-t5-xxl", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
-    pytest.param("aws-neuron/Mistral-7B-Instruct-v0.1-neuron-1x2048-24-cores", None, "ml.inf2.48xlarge", marks=pytest.mark.inf2),
+    pytest.param("HuggingFaceH4/zephyr-7b-beta", None, "ml.inf2.8xlarge", marks=pytest.mark.inf2),
```

Comment:
Could we please not remove 
```
    pytest.param("aws-neuron/Mistral-7B-Instruct-v0.1-neuron-1x2048-24-cores", None, "ml.inf2.48xlarge", marks=pytest.mark.inf2),
```
and just add on?

Reply:
As per `0.0.19` we don't maintain the specific neuron models, so there won't be any compatible model at `aws-neuron/Mistral-7B-Instruct-v0.1-neuron-1x2048-24-cores`. This is why I changed it to use a standard model that will be exported from cached artifacts instead.

Reply:
I see thanks for providing reasonings.